### DISCUSSION TITLE
component q-btn-dropdown not works well in safari

### DISCUSSION THREAD

**Discussion by pengtech** [quasarframework/quasar#13836] [CLI - BEX mode]
hi, we have an internal project developing with quasar, all things runs OK in chrome. Recently, we  checked and tested the compatibility of browser, found all **q-btn-dropdown** components cannot display well in safari(13.1.3 (15609.4.1)):

`
<q-btn-dropdown label="project">
  <q-list>
        <q-item clickable v-close-popup @click="onItemClick">
          <q-item-section>
            <q-item-label>Photos</q-item-label>
          </q-item-section>
        </q-item>
  </q-list>
</q-btn-dropdown>
`
the exact issue is label "project" is not displayed in web browser of safari,  but others like 'dropdown list', 'dropdown icon' display   normal. so weird.

I try to resolve this issue by updating quasar to the lastest version, by restarting project......, all attempts failed.
 hope get helps out there 

<img width="225" alt="aaaa" src="https://user-images.githubusercontent.com/38747593/176877684-099b4137-83f2-424d-8f53-223321ef4f22.png">

**Comment by pdanpdan** [comment]
please make a codepen that can reproduce this when using safari (mac safari?)

**Reply by pengtech** [reply]
yes, mac safari
